docs/content.zh/release-notes/flink-1.20.md

```diff
@@ -0,0 +1,433 @@
+`state.checkpoints.file-merging.enabled` to `true`. For more advanced options and principle behind
+this feature, please refer to the document of `Checkpointing`.
+
+#### Reorganize State & Checkpointing & Recovery Configuration
+
+##### [FLINK-34255](https://issues.apache.org/jira/browse/FLINK-34255)
+
+Currently, all the options about state and checkpointing are reorganized and categorized by
+prefixes as listed below:
+
+1. `execution.checkpointing.*`: all configurations associated with checkpointing and savepoint.
+2. `execution.state-recovery.*`: all configurations pertinent to state recovery.
+3. `state.*`: all configurations related to the state accessing.
+    1. `state.backend.*`: specific options for individual state backends, such as RocksDB.
+    2. `state.changelog.*`: configurations for the changelog, as outlined in FLIP-158, including the options for the "Durable Short-term Log" (DSTL).
+    3. `state.latency-track.*`: configurations related to the latency tracking of state access.
+
+In the meantime, all the original options scattered everywhere are annotated as `@Deprecated`.
+
+#### Use common thread pools when transferring RocksDB state files
+
+##### [FLINK-35501](https://issues.apache.org/jira/browse/FLINK-35501)
+
+The semantics of `state.backend.rocksdb.checkpoint.transfer.thread.num` changed slightly:
+If negative, the common (TM) IO thread pool is used (see `cluster.io-pool.size`) for up/downloading RocksDB files.
+
+#### Expose RocksDB bloom filter metrics
+
+##### [FLINK-34386](https://issues.apache.org/jira/browse/FLINK-34386)
+
+We expose some RocksDB bloom filter metrics to monitor the effectiveness of bloom filter optimization:
+
+`BLOOM_FILTER_USEFUL`: times bloom filter has avoided file reads.
+`BLOOM_FILTER_FULL_POSITIVE`: times bloom FullFilter has not avoided the reads.
+`BLOOM_FILTER_FULL_TRUE_POSITIVE`: times bloom FullFilter has not avoided the reads and data actually exist.
+
+#### Manually Compact Small SST Files
+
+##### [FLINK-26050](https://issues.apache.org/jira/browse/FLINK-26050)
+
+In some cases, the number of files produced by RocksDB state backend grows indefinitely.
+This might cause task state info (TDD and checkpoint ACK) to exceed RPC message size and fail recovery/checkpoint
+in addition to having lots of small files.
+
+In Flink 1.20, you can manually merge such files in the background using RocksDB API.
+
+### Runtime & Coordination
+
+#### Support Job Recovery from JobMaster Failures for Batch Jobs
+
+##### [FLINK-33892](https://issues.apache.org/jira/browse/FLINK-33892)
+
+In 1.20, we introduced a batch job recovery mechanism to enable batch jobs to recover as much progress as possible 
+after a JobMaster failover, avoiding the need to rerun tasks that have already been finished.
+
+More information about this feature and how to enable it could be found in: https://nightlies.apache.org/flink/flink-docs-master/docs/ops/batch/recovery_from_job_master_failure/
```

Comment:
What about using the link to the 1.20 documentation instead of `flink-docs-master`? The docs on master keeps changing and I'm afraid the content might be changed in the future.